manim/utils/ipython_magic.py

```diff
@@ -137,12 +137,5 @@ def construct(self):
                 )
 
 
-def _video_hash(path):
-    sha1 = hashlib.sha1()
-    with open(path, "rb") as f:
-        while True:
-            data = f.read(65536)
-            if not data:
-                break
-            sha1.update(data)
-    return sha1.hexdigest()
+def _generate_file_name():
+    return config["scene_names"][0] + "_" + datetime.now().strftime("%Y-%m-%d_%H-%M-%S")
```

Comment:
```suggestion
    return config["scene_names"][0] + "@" + datetime.now().strftime("%Y-%m-%d@%H-%M-%S")
```

Reply:
I don't know if this line ""%Y-%m-%d@%H-%M-%S")" might break something, but I think it should work

Reply:
> I don't know if this line ""%Y-%m-%d@%H-%M-%S")" might break something, but I think it should work

Yea I didn't see the hash being used anywhere besides for naming the files so don't think it should break anything

Reply:
Great! then I think this is good to go!